web/src/layout/home/index.tsx

```diff
@@ -147,8 +147,8 @@ const HomeView = (props: Props) => {
             <div className="d-flex flex-row align-items-center justify-content-center flex-wrap">
               <ExternalLink
                 className={`btn btn-secondary mb-4 mb-md-2 ${styles.socialBtn}`}
-                href="https://github.com/cncf/hub"
-                label="Open Github link"
+                href="https://github.com/artifacthub/hub"
+                label="Open GitHub link"
```

Comment:
This is why I filed this PR. Specifically, the links to https://github.com/cncf/hub on https://artifacthub.io/

Reply:
Looks ok, thanks

Reply:
`https://github.com/artifacthub/hub` is ok, thanks